service/src/main/java/datawave/microservice/authorization/AuthorizationOperationsV1.java

```diff
@@ -98,10 +122,32 @@ public AuthorizationsListBase<?> listEffectiveAuthorizations(@AuthenticationPrin
         return list;
     }
     
+    public GenericResponse<String> flushCachedCredentials(ProxiedUserDetails currentUser, boolean federate) {
+        GenericResponse<String> response = new GenericResponse<>();
+        log.info("Flushing credentials for " + currentUser + " from the cache.");
+        
+        // if we have any remote services configured, then flush those credentials as well
+        if (federate) {
+            for (UserOperations federatedUserOperations : registeredFederatedUserOperations) {
+                try {
+                    federatedUserOperations.flushCachedCredentials(currentUser);
+                } catch (Exception e) {
+                    log.error("Failed to flush user from remote user service", e);
+                    response.addMessage("Unable to user from remote user service");
```

Comment:
Should this unable to flush?
```suggestion
                    response.addMessage("Unable to flush user from remote user service");
```

Reply:
yes.  good catch.